website/src/components/header.tsx

```diff
@@ -0,0 +1,184 @@
+              <a key={item.name} href={item.href}>
+                <span className="sr-only">{item.name}</span>
+                <img src={item.icon} className="icon" />
+              </a>
+            ))}
+            <iframe
+              src="https://ghbtns.com/github-btn.html?user=functionless&repo=functionless&type=star&count=true&size=small"
+              frameBorder={0}
+              scrolling="0"
+              width="150"
+              height="20"
+              title="GitHub"
+            ></iframe>
+          </div>
+        </div>
+      </div>
+
+      <Transition
+        as={Fragment}
+        enter="duration-200 ease-out"
+        enterFrom="opacity-0 scale-95"
+        enterTo="opacity-100 scale-100"
+        leave="duration-100 ease-in"
+        leaveFrom="opacity-100 scale-100"
+        leaveTo="opacity-0 scale-95"
+      >
+        <Popover.Panel
+          focus
+          className="absolute inset-x-0 top-0 origin-top-right transform p-2 transition md:hidden"
+        >
+          <div className="divide-y-2 divide-gray-50 rounded-lg bg-functionless-white dark:bg-functionless-black shadow-lg ring-1 ring-black ring-opacity-5">
+            <div className="px-5 pt-5 pb-6">
+              <div className="flex items-center justify-between">
+                <div>
+                  <img
+                    className="h-8 w-auto"
+                    src="/img/logo/light.svg"
+                    alt="Your Company"
+                  />
+                </div>
+                <div className="-mr-2">
+                  <Popover.Button className="inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-100 hover:text-gray-500 focus:outline-none ring-0 border-none bg-transparent">
+                    <span className="sr-only">Close menu</span>
+                    <XMarkIcon className="h-6 w-6" aria-hidden="true" />
+                  </Popover.Button>
+                </div>
+              </div>
+              <div className="mt-6">
+                <nav className="grid gap-6">
+                  {[].map((item) => (
+                    <a
+                      key={item.name}
+                      href={item.href}
+                      className="-m-3 flex items-center rounded-lg p-3 hover:bg-gray-50"
+                    >
+                      <div className="flex h-10 w-10 flex-shrink-0 items-center justify-center rounded-md bg-indigo-500 text-white">
+                        <item.icon className="h-6 w-6" aria-hidden="true" />
+                      </div>
+                      <div className="ml-4 text-base font-medium text-gray-900">
+                        {item.name}
+                      </div>
+                    </a>
+                  ))}
+                </nav>
+              </div>
+            </div>
+            <div className="py-6 px-5">
+              <div className="grid grid-cols-2 gap-4">
+                {[].map((item) => (
```

Comment:
map from an empty list?

Reply:
Will move custom layout to docusaurus layout and utilise config to render items here